src/component/Meals.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+// import meal from '../Redux/meal';
+
+function Meals() {
+  const meals = useSelector((state) => state.meal.meals);
+  const isLoading = useSelector((state) => state.meal.loading);
+  const error = useSelector((state) => state.meal.error);
+  const [search, setSearch] = useState('');
+  function handleSearch(e) {
+    setSearch(e.target.value);
+  }
+  if (isLoading) {
+    return <p>Please wait...</p>;
+  }
+  if (error) {
+    return <p>{error}</p>;
+  }
+  return (
```

Comment:
- [ ] Consider choosing one color fot the text in your design so as to have a uniform color design as is in the given design.

Your design
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/0d1c9a7c-306f-4145-8070-129d0cdac32a)

Given design
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/3bf52967-2c37-4f1a-bd98-7df8cd03647d)

- Also follow the font-weight as this is a core requirement. Notice the font-weight difference between your text and the given text.